chameleonultragui/lib/gui/connectpage.dart

```diff
@@ -92,6 +92,7 @@ class ConnectPage extends StatelessWidget {
                           ...result.map<Widget>((chameleonDevice) {
                             return ElevatedButton(
                               onPressed: () async {
+                                // TODO: move this to gui/flashing.dart
```

Comment:
No, it will stay in place

Reply:
To me this functionally seems pretty bad UX, cause if a user does not want to update the firmware then atm nothing happens. Maybe we could make a second flash page then cause I would like that a user can just choose between using online or zip etc. And we need such a page for web already because we cannot just reconnect to a device in dfu mode.

Reply:
There is no exit DFU method afaik, so nothing we can do from GUI side

Reply:
I have resolved my concerns by adding a specific card on the connect page when the device is in DFU mode so users can choose how they want to flash the device